test/test_versioned_nodes.py

```diff
@@ -4,9 +4,9 @@
 import unittest
 
 host = "localhost"
-user = "test"
-password = "test"
-database = "postgres"
+user = "postgres"
+password = "postgres"
+database = "gen3datamodel_test"
```

Comment:
can we set those in the conftest (since they're also used in `db_config`) or a utils file instead of duplicating in each test file?

Reply:
Yes, good idea!